documentation/aks-upgrade.md

```diff
@@ -94,14 +94,22 @@ az aks nodepool get-upgrades --resource-group <ResourceGroup> --cluster-name <Cl
     - Change kubernetes version, plan and apply
     - Change default node pool orchestrator version, plan and apply
     - Change application node pool orchestrator version, plan and apply
-1. Follow the same manual process to upgrade the platform_test cluster
+    - Run following command's in another terminal to see the Cluster upgrade progress.
+        -  kubectl get events --watch
+        -  kubectl get nodes --watch
+3. Follow the same manual process to upgrade the platform_test cluster
     - Test with the welcome app: https://www.platform-test.teacherservices.cloud/
     - Raise PR
     - Wait 24h
-1. Follow the same manual process to upgrade the test cluster
+4. Follow the same manual process to upgrade the test cluster
     - Test with the welcome app: https://www.test.teacherservices.cloud/
     - Raise PR
     - Wait 24h
-1. Follow the same manual process to upgrade the production cluster
+5. Follow the same manual process to upgrade the production cluster
     - Test with the welcome app: https://www.teacherservices.cloud/
     - Raise PR
+
+## Troubleshooting
+
+1. If you see any failures , Login Azure portal and Go to AKS Cluster.
```

Comment:
Typos: 
If you see any failures, login to the Azure portal and go to the AKS cluster.